.github/workflows/documentation.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
```

Comment:
A bit confused by this one – I don't think there is a v4 version of the `actions/checkout`. Should it be v3?

Reply:
Yes, that's a mistake!